board.go

```diff
@@ -117,6 +117,8 @@ type BoardConfigurationColumnConfig struct {
 type BoardConfigurationColumn struct {
 	Name   string                           `json:"name"`
 	Status []BoardConfigurationColumnStatus `json:"statuses"`
+	Min    int                              `json:"min"`
```

Comment:
Please add the `omitempty` tag to the json config for these new fields as well as tests with omitted min/max

Reply:
Thank you for the quick reply! Made the changes. 😀